packages/react-components/src/organisms/__tests__/WorkingGroupCard.test.tsx

```diff
@@ -49,3 +50,12 @@ it('renders the working group shortText linking to the working group', () => {
     expect.stringMatching(/42$/),
   );
 });
+
+it('renders the state tag for a complete working group', () => {
+  const { getByText, rerender, queryByText } = render(
+    <WorkingGroupCard {...props} complete={true} />,
+  );
+  expect(getByText('Complete', { selector: 'span' })).toBeVisible();
+  rerender(<WorkingGroupCard {...props} complete={false} />);
+  expect(queryByText('Complete')).not.toBeInTheDocument();
```

Comment:
Probally should have checked card background but its minor

Reply:
nice! I've added the background color check in here https://github.com/yldio/asap-hub/pull/2364/commits/651c21aa3ddc1d46b92ccb567ddc2c1ffc55d6d2 :)